Ports/README.md

```diff
@@ -358,7 +359,7 @@ run_replace_in_file "s/define FOO 1/undef FOO/" config.h
 
 You can either:
 
-- Add new ports - just get the software to build and add the necessary patches
+- Add new ports – just get the software to build and add the necessary patches
```

Comment:
useless noise, there's hundreds of these across the project - fixing one is not helpful.

Reply:
well… 🤷‍♂️